stdlib/contrib/tomhollingworth/events/duration.flux

```diff
@@ -1,16 +1,98 @@
+// Package events provides tools for analyzing event-based data.
+//
+// introduced: 0.91.0
 package events
 
 
-// duration will calculate the duration between records
-// for each record. The duration calculated is between
-// the current record and the next. The last record will
-// compare against either the stopColum (default: _stop)
-// or a stop timestamp value.
-//
-// `timeColumn` - Optional string. Default '_time'. The value used to calculate duration
-// `columnName` - Optional string. Default 'duration'. The name of the result column
-// `stopColumn` - Optional string. Default '_stop'. The name of the column to compare the last record on
-// `stop` - Optional Time. Use a fixed time to compare the last record against instead of stop column.
+// duration calculates the duration of events.
+//
+// The function determines the time between a record and the subsequent record
+// and associates the duration with the first record (start of the event).
+// To calculate the duration of the last event,
+// the function compares the timestamp of the final record
+// to the timestamp in the stopColumn or the specified stop time.
+//
+// > ## Similar functions
+// > events.duration() is similar to elapsed() and stateDuration(), but differs in important ways:
+// >
+// >elapsed() drops the first record. events.duration() does not.
+// >stateDuration() calculates the total time spent in a state (determined by a predicate function). events.duration() returns the duration between all records and their subsequent records.
+// >
+// >For examples, see below.
```

Comment:
- This stuff will just get dropped into the JSON output as raw markdown, so we need to format code as code.
- We won't be using blockquote styling for notes in the generated docs.

```suggestion
// ### Similar functions
// `events.duration()` is similar to `elapsed()` and `stateDuration()`, but differs in important ways:
// 
// - `elapsed()` drops the first record. `events.duration()` does not.
// - `stateDuration()` calculates the total time spent in a state (determined by a predicate function).
//   `events.duration()` returns the duration between all records and their subsequent records.
// 
// See the example [below](#compared-to-similar-functions).
```